src/components/Firestore/FirestoreEmulatedApiProvider.tsx

```diff
@@ -119,7 +119,10 @@ export function useSubCollections(
   docRef: firebase.firestore.DocumentReference
 ) {
   const { baseUrl } = useFirestoreRestApi();
-  const encodedPath = docRef.path; // TODO: Encode each segment
+  const encodedPath = docRef.path
+    .split('/')
+    .map(uri => encodeURIComponent(uri.replace(/%/g, '')))
```

Comment:
There is no need to remove % here -- the Firestore Emulator, unlike React router, handles it just fine.